test/integration/manifests/datapath/windows-deployment.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: windows-pod
+  namespace: datapath-win
+spec:
+  replicas: 4
+  selector:
+    matchLabels:
+      app: datapod
+  template:
+    metadata:
+      labels:
+        app: datapod
+    spec:
+      containers:
+        - name: windows-container
+          image: mcr.microsoft.com/dotnet/framework/samples:aspnetapp
```

Comment:
Sorry I missed this in the initial review. Can we use a smaller image ? The size is way aroung 1.5 Gb. 
This one: mcr.microsoft.com/windows/nanoserver:ltsc2022 size is around 100mb, but not sure if it resolves all the cmds you want to execute.